lib/GenomeFileUtil/core/GenomeToGFF.py

```diff
@@ -153,7 +153,14 @@ def feature_sort(feat):
         file_handle = open(out_file_path, 'w')
         writer = csv.DictWriter(file_handle, gff_header, delimiter="\t",
                                 escapechar='\\', quotechar="'")
-        for contig in genome_data.get('contig_ids', features_by_contig.keys()):
+
+        contig_list = list()
+        # fix for empy list of contig_ids
+        if "contig_ids" in genome_data and len(genome_data["contig_ids"]) > 0:
```

Comment:
looks good, this could be `genome_data.get('contig_ids')` for brevity but this is probably more clear

Reply:
Just FYI slebras, it was that and that caused problems. When the list was empty but the key existed, it would print out nothing.